src/condor_dagman/parse.cpp

```diff
@@ -1475,8 +1472,7 @@ parse_abort(
 	}
 
 		// RETURN keyword.
-	bool haveReturnVal = false;
-	int returnVal = 9999; // assign value to avoid compiler warning
+	int returnVal = INT_MAX; // assign value to avoid compiler warning
```

Comment:
We should always use something like `std::numeric_limits<int>::max` instead of these C macros like `INT_MAX`